src/containers/DAO/pages/MintDAO/MintDAO.types.ts

```diff
@@ -0,0 +1,27 @@
+export enum Step {
+	None = 'None',
+	Unlock = 'Unlock',
+	Mint = 'Mint',
+}
+
+export enum StepStatus {
+	Normal = 'Normal',
+	Confirm = 'Confirm',
+	Processing = 'Propcessing',
+}
+
+export enum ErrrorType {
+	Signature = 'Signature',
+	Transaction = 'Transaction',
```

Comment:
This flow won't be generating any signatures, so can remove the `Transaction` error type